docs/csharp/language-reference/keywords/fixed-statement.md

```diff
@@ -19,6 +19,12 @@ You can initialize a pointer by using an array, a string, a fixed-size buffer, o
 
 [!code-csharp[Initializing fixed size buffers](../../../../samples/snippets/csharp/keywords/FixedKeywordExamples.cs#2)]
 
+Starting with C# 7.3, the `fixed` statement operates on additional types beyond arrays, strings, fixed-size buffers, or unmanaged variables. Any type that implements a method named `DangerousGetPinnableReference` can be pinned. The `DangerousGetPinnableReference` must return a `ref` variable to an unmanaged type. See the topic on [pointer types](../../programming-guide/unsafe-code-pointers/pointer-types.md) for more information. The .NET Types <xref:System.Span%601?displayProperty=nameWithType>, <xref:System.ReadonlySpan%601?displayProperty=nameWithType> introduced in .NET Standard 2.0 make use of this pattern and can be pinned. This is shown in the following example:
```

Comment:
- .NET Types --> .NET types
- "and" rather than comma between Span\<T> and ReadOnlySpan\<T>
- I don't think that they're part of [.NET Standard 2.0](https://raw.githubusercontent.com/dotnet/standard/master/docs/versions/netstandard2.0_diff.md). I think they're part of .NET Core 2.x.